packages/ember-glimmer/tests/integration/components/curly-components-test.js

```diff
@@ -1968,52 +1968,6 @@ moduleFor('Components test: curly components', class extends RenderingTest {
     assert.equal(outer.parentView, this.context, 'x-outer receives the ambient scope as its parentView');
   }
 
-  ['@test component should share the viewRegistry'](assert) {
```

Comment:
Removed this test, since it is asserting a thing that is actually not valid (we do not want a `_viewRegistry` set on view/component instances at all).